dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -86,6 +86,9 @@ public Entry parseWorkflowContent(Entry entry, String filepath, String content,
                                 description = first.get().getContent();
                             }
                         }
+                    } else if (doc instanceof List) {
+                        List docList = (List)doc;
+                        description = String.join(System.getProperty("line.separator"), docList);
```

Comment:
This triggers a thought... When a remote entry is run with the Dockstore CLI, I assume it always has the line.separator of where the Dockstore server is running. So in theory it's possible that the line.separator on the CLI and the server may not match.

Really an extreme edge case that I'd never thought of before; I can log it separately if you think it's a legit issue.

Reply:
When running the remote entry, the description is entirely irrelevant.